mlibc-config.h.in

```diff
@@ -0,0 +1,8 @@
+
+#mesondefine __MLIBC_ANSI_OPTION
+#mesondefine __MLIBC_POSIX_OPTION
+#mesondefine __MLIBC_LINUX_OPTION
+#mesondefine __MLIBC_INTL_OPTION
+#mesondefine __MLIBC_ICONV_OPTION
+#mesondefine __MLIBC_GLIBC_OPTION
+
```

Comment:
This file is missing an include guard and it has unnecessary newlines at the top and bottom.

Reply:
It follows the same style as ``internal-config.h.in``, I could also modify both to be more uniform.